Content.Client/StatusIcon/StatusIconOverlay.cs

```diff
@@ -42,8 +38,6 @@ protected override void Draw(in OverlayDrawArgs args)
         var scaleMatrix = Matrix3.CreateScale(new Vector2(1, 1));
         var rotationMatrix = Matrix3.CreateRotation(-eyeRot);
 
-        handle.UseShader(_shader);
```

Comment:
This might need to be looked at by an art director.

Reply:
Split this out.